arm-software/linux/CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [20.1.0]
+
+This is the first release of the Arm Toolchain for Linux (ATfL), a successor of
+the Arm Compiler for Linux.
+
+Although it is entirely based on the LLVM project version 20.1, there were some
+changes introduced specifically for the ATfL. Most notably, these were:
+
+- The compiler uses a config file by default, which improves
+  performance-specific optimizations; most notably, it encourages the use of the
+  vectorized mathematical routines in the Loop Vectorizer which enables the
+  possibility of vectorizing loops containing the calls to the mathematical
+  library functions.
+
+- For function whose `vscale_range` is limited to a single value, ATfL can size
+  scalable vectors.
```

Comment:
Should we add something about permitting bitcast between scalable and fixed.
```
Enable LLVM to perform bitcast-like operations between fixed and scalable vectors, improving optimization opportunities for code utilizing scalable vector types
```

Reply:
Added an item

Reply:
I think it should be part of this item.

Reply:
Integrated into one.